docking_task_node:
  ros__parameters:
    # map_origin_lat: 63.4411585867919
    # map_origin_lon: 10.419400373255625
    map_origin_lat: -33.72242824301795
    map_origin_lon: 150.6740063854522
    map_origin_set: true
    gps_start_lat: -33.72255346763595
    gps_start_lon: 150.67394210459224
    gps_end_lat: 63.44125901804796
    gps_end_lon: 10.41857835889424
    gps_start_x: 0.0
    gps_start_y: 0.0
    gps_end_x: 0.0
    gps_end_y: 0.0
    gps_frame_coords_set: false
    map_origin_topic: "/map/origin"
    odom_topic: "/seapath/odom/ned"
    landmark_topic: "landmarks_out"
    assignment_confidence: 10  # Number of consequtive identical assignments from auction algorithm before we consider the assignment as correct

    # Task specific parameters (not derived from base class)
    distance_to_first_buoy_pair: 2.0 # Distance in x-direction to first buoy pair in meters from current postition (not gps) position in base_link frame
    # If the distance is greater than 6.0 we drive distance-4m towards the first buoy pair before trying to find it
    distance_between_buoys_in_pair: 5.0 # Distance between the two buoys in a pair
    grid_topic: "pcl_grid"
    dock_structure_absolute_width: 4.39 # Width of the dock structure in meters 4.39
    dock_structure_absolute_width_tolerance: 0.5 # Padding on each side of the dock structure for line search start and end
    dock_edge_width: 0.13   # width of the dock edge
    line_search_distance: 2.0 # distance to iterate along line when searching for dock edge
    dock_search_offset: 2.0   # offset in forward direction from third buoy pair to begin search for dock
    




    dock_width: 4.13 # width of available docking space
    dock_width_tolerance: 0.5
    dock_length: 2.0  # length of available docking space
    dock_length_tolerance: 0.5
    dock_edge_width_tolerance: 0.05
    task_nr: 2 # possible values 1 and 2 corresponding to task 4.1 and 4.2
    models:
      dynmod_stddev: 0.01
      senmod_stddev: 0.01
    